src/pool/cli.c

```diff
@@ -2001,6 +2001,11 @@ attr_check_input(int n, char const *const names[], void const *const values[],
 
 			return -DER_INVAL;
 		}
+		if (strlen(names[i]) > DAOS_ATTR_NAME_MAX) {
```

Comment:
can use strnlen here?

Reply:
yes, you are right again ... ;-)

Reply:
there is also another occurence in container/cli.c just in case you don't miss that too.